### PR TITLE
Fixed layout issue on repository panel

### PR DESCRIPTION
**Pull request by enicolasgomez**
Changed repository browser's layout to absolute position with a z-index = 100 so it overlays the canvas without replacing it

### CODE REVIEW

File: src/test-app/app.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ const fileExtensionRe = /(?:\.([^.]+))?$/;
         <!--Views-->
 
         <section id="main-panel">
-            <section id="repo-panel" *ngIf="showRepoPanel" class="w3-quarter w3-gray w3-border-right w3-border-white">
+            <section id="repo-panel" *ngIf="showRepoPanel" class="w3-quarter w3-gray w3-border-right w3-border-white" style="position:absolute; z-index: 100;">
```
Comment (zsinnema) [inline src/test-app/app.js:82]: we should use style on an element... please move it to the css file

Comment (zsinnema) [reply src/test-app/app.js:82]: also I don't like the vertical scrollbarr appearing when the list is populated

**Review by zsinnema — CHANGES_REQUESTED** [review]

File: src/test-app/app.js (hunk 1 above)
Comment (zsinnema) [reply src/test-app/app.js:82]: imo the issue should be fixed on the main-panel and not in the repo-panel:
```
#main-panel[_ngcontent-ukd-c0] mat-tab-group[_ngcontent-ukd-c0] {
    height: inherit;
    /* width: calc(100%); */
}
```
by removing/commenting the width

Comment (enicolasgomez) [reply src/test-app/app.js]: Changed and pushed as requested. Please verify.

**Review by zsinnema — APPROVED** [review]

**Pull request merged**